{
	"document_title": "CVE-2022-29228",
	"document_type": "SUSE CVE",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "SUSE CVE-2022-29228"
		},
		"status": "Interim",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "5",
				"date": "2024-06-04T23:33:50Z",
				"description": "current"
			}
		},
		"initial_release_date": "2022-06-10T23:47:15Z",
		"current_release_date": "2024-06-04T23:33:50Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf-cve.pl",
			"date": "2020-12-27T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "CVE-2022-29228",
			"title": "CVE",
			"type": "Summary"
		},
		{
			"text": "Envoy is a cloud-native high-performance proxy. In versions prior to 1.22.1 the OAuth filter would try to invoke the remaining filters in the chain after emitting a local response, which triggers an ASSERT() in newer versions and corrupts memory on earlier versions. continueDecoding() shouldnât ever be called from filters after a local reply has been sent. Users are advised to upgrade. There are no known workarounds for this issue.",
			"title": "Mitre CVE Description",
			"type": "Description"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		}
	],
	"document_references": [
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		}
	],
	"product_tree": {},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "Envoy is a cloud-native high-performance proxy. In versions prior to 1.22.1 the OAuth filter would try to invoke the remaining filters in the chain after emitting a local response, which triggers an ASSERT() in newer versions and corrupts memory on earlier versions. continueDecoding() shouldnât ever be called from filters after a local reply has been sent. Users are advised to upgrade. There are no known workarounds for this issue.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2022-29228",
			"threats": [
				{
					"type": "Impact",
					"description": "important"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "5",
					"vector_v_2": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
				},
				"score_set_v_3": {
					"base_score_v_3": "7.5",
					"vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				}
			}
		}
	]
}
